extensions/map-layers-auth/src/ArcGis/ArcGisAccessClient.ts

```diff
@@ -287,7 +287,12 @@ export class ArcGisAccessClient implements MapLayerAccessClient {
 
     const endpointStr = (endpointType === ArcGisOAuth2EndpointType.Authorize ? "authorize" : "token");
     const urlObj = new URL(url);
-    if (urlObj.hostname.toLowerCase().endsWith("arcgis.com")) {
+
+    const allowedHosts = [
+      "arcgis.com"
+    ];
+
+    if (allowedHosts.includes(urlObj.hostname.toLowerCase())) {
```

Comment:
I don't think this will work.
We want to check host names such as: services7.arcgis.com

Reply:
changed and added test